decidim-admin/app/packs/entrypoints/decidim_admin.js

```
@@ -27,6 +26,6 @@ import "src/decidim/admin/taxonomy_filters"
 import "entrypoints/decidim_admin.scss";
 
 document.addEventListener("turbo:load", () => {
-  initLanguageChangeSelect(document.querySelectorAll("select.language-change"));
+  document.querySelectorAll("select.language-change").forEach((container) =>
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)